### PR TITLE
WIP: Refactored ingestion to database code

### PR DESCRIPTION
**Pull request by GeetikaBatra**

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -82,23 +90,92 @@ def validate_request_data(input_json):
...
+                    }
+        return return_dict
+
+
+class DatabaseIngestion():
+
+    
+
+    @staticmethod
+    def _update_data(session, data):
```
Comment (samuzzal-choudhury) [inline src/utils.py]: @GeetikaBatra looks like the cls parameter is missing from this parameters list.

File: src/utils.py [hunk 2]
```diff
@@ -82,23 +90,92 @@ def validate_request_data(input_json):
...
+        return_dict = { OSIORegisteredRepos.git_url:data["git_url"],
+                        OSIORegisteredRepos.git_sha:data["git_sha"],
+                        OSIORegisteredRepos.email_ids:data["email_ids"],
+                        OSIORegisteredRepos.last_scanned_at:datetime.datetime.now()
+                    }
+        return return_dict
+
+
+class DatabaseIngestion():
+
```
Comment (samuzzal-choudhury) [inline src/utils.py]: @GeetikaBatra please run `check-docstyle.sh` and `run-linter.sh` locally to ensure code sanity.

File: src/utils.py (hunk 1 above)
Comment (GeetikaBatra) [reply src/utils.py]: it's a static method, cls parameter is not required here.

File: src/utils.py (hunk 2 above)
Comment (GeetikaBatra) [reply src/utils.py]: sure, on it.

File: src/utils.py (hunk 1 above)
Comment (samuzzal-choudhury) [reply src/utils.py]: ok. I missed that

**Pull request merged**